packages/sindarian-ui/src/globals.css

```
@@ -328,6 +328,7 @@
 /* Sonner toast overrides — solid bold backgrounds (Radix parity) */
 [data-sonner-toaster] {
   --width: 420px;
+  pointer-events: auto !important;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)